include/nlohmann/json.hpp

```diff
@@ -4943,6 +4943,23 @@ class basic_json
         return {it, res.second};
     }
 
+    /// helper for insertion of an iterator (supports GCC 4.8+)
```

Comment:
Could you please reference this PR in a comment, so one may find the rationale for this change quicker?